src/khoj/processor/tools/online_search.py

```diff
@@ -96,18 +98,24 @@ async def search_online(
         yield response_dict
         return
 
-    logger.info(f"🌐 Searching the Internet for {subqueries}")
-    if send_status_func:
-        subqueries_str = "\n- " + "\n- ".join(subqueries)
-        async for event in send_status_func(f"**Searching the Internet for**: {subqueries_str}"):
-            yield {ChatEvent.STATUS: event}
-
-    if SERPER_DEV_API_KEY:
+    if GOOGLE_SEARCH_API_KEY and GOOGLE_SEARCH_ENGINE_ID:
+        search_engine = "Google"
+        search_func = search_with_google
+    elif SERPER_DEV_API_KEY:
+        search_engine = "Serper"
         search_func = search_with_serper
     elif JINA_API_KEY:
+        search_engine = "Jina"
         search_func = search_with_jina
     else:
-        search_func = search_with_searxng
+        search_engine = "Jina"
```

Comment:
This doesn't work without an API key anymore, so let's skip if none of the options are configured?

Reply:
Oh yeah, fixed to fallback to searxng by default (not Jina)